app/src/main/res/values-it-rIT/strings.xml

```diff
@@ -2,89 +2,89 @@
-    <string name="setup_mod_info_action">Start mod download</string>
-    <string name="setup_mod_oneui_title">OneUI Detected</string>
-    <string name="setup_mod_oneui_content">This device appears to be running OneUI, which prevents the SmartThings Mod from working. uTag is not compatible with OneUI without root, please use the official SmartThings app instead.\n\n\<b>Note:</b> If your device is rooted with Xposed installed, make sure SmartThings is set up, enable the uTag Xposed module, force stop SmartThings and return to uTag to continue.</string>
-    <string name="setup_mod_downloading">Downloading SmartThings…</string>
-    <string name="setup_mod_setting_up">Setting up mod…</string>
-    <string name="setup_mod_setting_up_subtitle">This may take a moment</string>
-    <string name="setup_mod_uninstall_title">Uninstall SmartThings</string>
-    <string name="setup_mod_uninstall_content">Since the modified version of SmartThings has a different signature, you must uninstall the currently installed version to continue.\n\n<b>Note:</b> This will clear SmartThings data from your device, requiring you to sign in again.</string>
-    <string name="setup_mod_uninstall_action">Uninstall</string>
-    <string name="setup_mod_install_title">Ready to install</string>
-    <string name="setup_mod_install_content">The modified version of SmartThings is ready to install. You may need to grant uTag permission to install packages to install the mod.</string>
-    <string name="setup_mod_install_action">Install</string>
-    <string name="setup_mod_setup_title">Set up SmartThings</string>
-    <string name="setup_mod_setup_content">Open SmartThings, grant its permissions, sign in and return to uTag to continue the uTag setup process.</string>
-    <string name="setup_mod_setup_action">Open SmartThings</string>
-    <string name="setup_mod_error_title">Error</string>
-    <string name="setup_mod_error_failed_to_get">Failed to load the latest version of the SmartThings mod from the server. Check your connection and try again later.</string>
-    <string name="setup_mod_error_failed_to_download">Failed to download the SmartThings mod. Check your connection and try again later.</string>
-    <string name="setup_mod_error_close">Close</string>
-    <string name="setup_mod_metered_dialog_title">Large Download Warning</string>
-    <string name="setup_mod_metered_dialog_content">The SmartThings mod APK file is %1s. Continue to download the file?</string>
+    <string name="setup_mod_info_content">Per lavorare con uTag e bypassare i requisiti di Samsung, è necessaria una versione modificata di SmartThings. uTag ti guiderà attraverso il download e l\'installazione.\n\n\<b>Nota:</b> Se il dispositivo è rootato con Xposed installato, assicurati che SmartThings sia configurato, abilita il modulo uTag Xposed, forza l\'arresto di SmartThings e torna a uTag per saltare questo passaggio.</string>
```

Comment:
`funzionare` sounds a little more natural in this context compared to `lavorare`.

```suggestion
    <string name="setup_mod_info_content">Per funzionare con uTag e bypassare i requisiti di Samsung, è necessaria una versione modificata di SmartThings. uTag ti guiderà attraverso il download e l\'installazione.\n\n\<b>Nota:</b> Se il dispositivo è rootato con Xposed installato, assicurati che SmartThings sia configurato, abilita il modulo uTag Xposed, forza l\'arresto di SmartThings e torna a uTag per saltare questo passaggio.</string>
```